SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/HoloNetViewModel.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Webhook;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition.ViewModel
+{
+    public class HoloNetViewModel : GuiViewModelBase<HoloNetViewModel, GuiPayloadBase>
+    {
+        public const int MaxHoloNetTextLength = 800;
+
+        protected override void Initialize(GuiPayloadBase initialPayload)
+        {
+            HoloNetText = string.Empty;
+            WatchOnClient(model => model.HoloNetText);
+        }
+        public string HoloNetText
+        {
+            get => Get<string>();
+            set
+            {
+                Set(value);
+            }
+        }
+
+        public Action OnClickSubmit() => () =>
+        {
+            if (string.IsNullOrWhiteSpace(HoloNetText))
+            {
+                return;
+            }
+
+            var message = HoloNetText;
+
+            if (message.Length > 600)
+            {
+                SendMessageToPC(Player, "Your HoloNet broadcast was too long. Please shorten it to no longer than 600 characters and resubmit the broadcast. For reference, your message was: \"" + message + "\"");
+                return;
+            }
+            var area = GetArea(Player);
+            var position = GetPosition(Player);
+
+            var url = Environment.GetEnvironmentVariable("SWLOR_HOLONET_WEBHOOK_URL");
```

Comment:
Is this intentionally a different environment variable? If so, please also add it to the swlor.env file.

Reply:
Discord tags webhook messages to different webhook bots. By giving them a different environment variable, it allows us to rename the bots differently; so messages from a HoloNet broadcast can be from a bot named "HoloNet broadcast" and messages from a DM broadcast can be from a different bot named "DM broadcast". Just a nice little differentiation IMO.

Reply:
I'm good with this. Thanks for adding the environment variables to swlor.env